R/contribution_matrix.R

```diff
@@ -627,9 +627,165 @@ CheckSingularMatrix <- function(matrix){
   return(contribution)
 }
 
+#' Create the contribution matrix in a convenient format.
+#' 
+#' @param data Input data in long format.
+#' @param covariate_title Title of covariate column in data.
+#' @param treatment_ids Data frame containing treatment IDs and names in columns named 'Number' and 'Label' respectively.
+#' @param outcome_type "Continuous" or "Binary".
+#' @param outcome_measure "MD", "OR", "RR" or "RD".
+#' @param effects_type "fixed" or "random".
+#' @param regression_coefficient_type Type of regression coefficient. One of: "Shared", "Unrelated", "Exchangeable"
```

Comment:
This is a duplicate of the parameter `cov_parameters`. Please use that and stick to lower case for consistency with other functions.